docs/DeveloperGuide.md

```diff
@@ -444,11 +458,11 @@ Priorities: MVP (must have), 2 (nice to have), 3 (unlikely to have)
       Use case resumes from Step 2.
 
 * **1b**. No contacts found matching the entered keyword.
-    * **1b1**. TutorEase displays a message: "No contacts found with the given keyword(s)."  
+    * **1b1**. TutorEase displays an error message, telling no contacts have been found with the given keyword.  
     * **1b2**. Tutor enters a new keyword or cancels the search.  
       Use case resumes from Step 2 or ends if cancelled.  
```

Comment:
Use case should resume from Step 1 if not cancelled.

See #304

Reply:
I will also add issue #304 under this PR

Reply:
Lets split this extension 1b into 2

1b2. Tutor enters the correct keyword
    Use case resume from Step 2
    
1c. No contacts found
    1c1. Display error message
    1c2. Cancel search
    Use case ends